src/styles/Blocks/product.scss

```diff
@@ -0,0 +1,120 @@
+.product {
+  display: block;
+  margin: 0;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 300ms;
+
+  &__photo {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+
+  &__code {
+    color: $secondary;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+
+    &-title {
+      line-height: 18px;
+      font-size: 12px;
+      color: $secondary;
+    }
+
+    &-cost {
+      line-height: 18px;
+      font-size: 16px;
+      color: $main-accent;
+    }
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+
+    &-active {
+      width: 166px;
+      height: 38px;
+
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      text-decoration: none;
+      font-size: 15px;
+      font-weight: 700;
+      color: $white;
+
+      background: $blue-accent;
+      border: 1px solid transparent;
+      border-radius: 5px;
+
+      transition: background-color 300ms;
+
+      cursor: pointer;
+    }
+
+    &-active:hover {
+      border: 1px solid $blue-accent;
+      color: $blue-accent;
+      background: $white;
+    }
+  }
+
+  &:hover{
+    transform: scale(1.2);
+  }
+
+  &:hover &__title{
+    color: #34568b;
+  }
+
+  &__stars-reviews {
+    display: flex;
+
+    margin-bottom: 24px;
+    align-items: center;
+    justify-content: space-between;
+
+      .stars {
+        display: flex;
+        gap: 4px;
+
+        &__star {
+          width: 16px;
+          height: 16px;
+          background: url(../images/star.svg) no-repeat center;
+        }
+
+        &__star:nth-child(-n + 4) {
+          background-image: url(../images/star-active.svg);
+        }
```

Comment:
What about other modifiers? It should work okay with different number of active stars

Reply:
If we talk about stars animation, as I know, we dont need it. I talk about it with other classmates, and they dont animate them. In my opinion, we should animate it with JS.
If i correctly understand the quastion..)